src/components/Layout/Home.tsx

```diff
@@ -52,6 +92,78 @@ const Home = () => {
           </Flex>
         </Center>
       </GridItem>
+      <GridItem
+        colSpan={1}
+        w="100%"
+        h="90vh"
+        padding="30"
+        position="relative"
+        borderRadius="20px">
+        <Flex
+          justifyContent="center"
+          alignItems="center"
+          position="relative"
+          top="0"
+          left="0"
+          width="100%"
+          height="100%">
+          <Box width="100%" maxWidth="100%" maxHeight="100%">
+            {images.map((image, index) => (
+              <Image
+                key={index}
+                src={image}
+                alt={`Slide ${index + 1}`}
+                opacity={index === currentSlide ? 1 : 0}
+                transition="opacity 1s ease-in-out"
+                position="absolute"
+                style={{ maxWidth: "100%", maxHeight: "100%" }}
+                top="10"
+                left="inherit"
+                objectFit="cover"
+                backgroundPosition="center"
+                borderRadius="20px"
+                border={"1px solid #d7da5e"}
+              />
+            ))}
+          </Box>
+        </Flex>
+      </GridItem>
+      <GridItem colSpan={1} w="100%" h="90vh">
+        <Flex flexDirection="column" position="relative">
+          <Box p={5}>
+            <Text fontSize="xl">Discover new recipes</Text>
+            <Text>
+              Olivier will find new delicious recipes for every taste. Say
+              goodbye to boring meals and hello to culinary inspiration. You can
+              easily add your own recipe, edit, adjust and store recipes.
```

Comment:
NON-BLOCKING: I might instead use:

`You can easily add, edit, adjust and store recipes.`